src/TrackGenerator.h

```diff
@@ -79,6 +79,9 @@ class TrackGenerator {
   /** Filename for the *.tracks input / output file */
   std::string _tracks_filename;
 
+  /** Filename suffix for the *.tracks input / output file */
+  std::string _tracks_filename_suffix;
```

Comment:
Yes - if I want to model the different clustered versions of the same geometry, with the same CMFD mesh and track laydown, I want the track files to be stored separately so I can actually take advantage of them for subsequent simulations. Perhaps even more important, I've noticed issues when running multiple simulations which write/load tracks files at the same time - it can cause one or more simulations to just hang.